src/crimson/common/config_proxy.cc

```diff
@@ -41,5 +41,9 @@ seastar::future<> ConfigProxy::start()
   });
 }
 
+void ConfigProxy::show_config(ceph::Formatter* f) const {
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes , in this case, it would be "crimson/common: ".

Reply:
done